helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/ticket/TicketService.java

```diff
@@ -28,16 +47,22 @@
 
   protected final TicketClient ticketClient;
 
+  protected final SymphonyClientUtil symphonyClientUtil;
+
+  protected final HelpDeskAi helpDeskAi;
+
   private final ValidateMembershipService validateMembershipService;
 
   public TicketService(SymphonyValidationUtil symphonyValidationUtil, SymphonyClient symphonyClient,
       HelpDeskBotConfig helpDeskBotConfig, TicketClient ticketClient,
-      ValidateMembershipService validateMembershipService) {
+      ValidateMembershipService validateMembershipService, HelpDeskAi helpDeskAi) {
```

Comment:
I believe this new parameter no longer required.

Reply:
Done.